pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook_suite.go

```diff
@@ -0,0 +1,195 @@
+package api
```

Comment:
Could we rename the file to `webhook_suitetest.go` to be similar to `kubebuilder/pkg/plugin/v3/scaffolds/internal/templates/config/controller/controller_suitetest.go`

Reply:
IMHO the `controller_suitetest.go` file name should be changed to `controller_suite_test.go`. The reason behind this the _test.go files are not imported when a package is imported. When I create a file `webhook_suitetest.go` and the package is imported in controllers, this test suite is also gets imported there and go test fails... 
```
?   	sigs.k8s.io/kubebuilder/testdata/project-v3	[no test files]
?   	sigs.k8s.io/kubebuilder/testdata/project-v3/api/v1	[no test files]
panic: You may only call BeforeSuite once!

goroutine 1 [running]:
github.com/onsi/ginkgo/internal/suite.(*Suite).SetBeforeSuiteNode(0xc0001046e0, 0x15be3e0, 0x18a0fe0, 0x22f1e0d, 0x63, 0x34, 0xc0003fa160, 0xac, 0xdf8475800)
	/home/prafull/go/pkg/mod/github.com/onsi/ginkgo@v1.12.1/internal/suite/suite.go:113 +0x170
github.com/onsi/ginkgo.BeforeSuite(0x15be3e0, 0x18a0fe0, 0xc00039c990, 0x1, 0x1, 0x14)
	/home/prafull/go/pkg/mod/github.com/onsi/ginkgo@v1.12.1/ginkgo_dsl.go:484 +0xab
sigs.k8s.io/kubebuilder/testdata/project-v3/controllers.init()
	/home/prafull/go/src/github.com/prafull01/kubebuilder/testdata/project-v3/controllers/suite_test.go:52 +0x75
FAIL	sigs.k8s.io/kubebuilder/testdata/project-v3/controllers	0.012s
FAIL
make: *** [Makefile:21: test] Error 1
```

I will raise another PR to change the `controller_suitetest.go` file name.

Reply:
Are you adding the webhooks in the controller path? The suggestion was regarded just to rename the files (boilerplate and the file that is generated) but not the path. The webhook_suitetest.go ought to be where the webhooks are scaffolded. 

So it would be to:
- Replace `webhook_suite_test.go` for `webhook_suitetest.go`
- Repalce `pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook_suite.go` for `pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook_suitetest.go`

Reply:
Yes.. I am scaffolding the `webhook_suitetest.go` in `api/v1/webhook_suitetest.go` only. However, controller suite is importing the v1 package like this `crewv1 "sigs.k8s.io/kubebuilder/testdata/project-v3/api/v1"` . This import is causing above issue mentioned in my previous comment. Any test suite we define should end `_test.go` so that the package imports doesn't imports those file and functions written in those file. Or we can have separate package like `v1_test` for the testing suites so that those are not imported when original package is imported.

1. Replace webhook_suite_test.go for webhook_suitetest.go : The file should end with `_test.go`, in controllers path the name of the file is simply `suite_test.go`. We can decide the naming convention here.

2. The name of `pkg/plugin/v3/scaffolds/internal/templates/config/api/webhook_suite.go`, I will replace it as per your suggestion.